test/uk/gov/hmrc/economiccrimelevyregistration/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -133,6 +125,13 @@ class CheckYourAnswersControllerSpec extends SpecBase {
           status(result)                                shouldBe SEE_OTHER
           session(result).get(SessionKeys.EclReference) shouldBe Some(createEclSubscriptionResponse.eclReference)
           redirectLocation(result)                      shouldBe Some(routes.RegistrationSubmittedController.onPageLoad().url)
+
+          verify(mockEmailService, times(1)).sendRegistrationSubmittedEmails(
```

Comment:
Do we need to wrap this in an 'eventually {}' like we did for the NRS test (otherwise don't we have a race condition)?

Reply:
Not in this case - because this is a mockito verify, not a wiremock verify. All mockito cares about is that the function was called, not what the result is or whether we waited for the result.

Reply:
Re: point 2, yes, I've updated